src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/entity/Entity.kt

```diff
@@ -151,6 +154,13 @@ open class Entity(val entity: MCEntity) {
      */
     fun getClassName(): String = entity.javaClass.simpleName
 
+    /**
+     * Gets the ModelBase of the entity
+     *
+     * @return the entity's main model
+     */
+    fun getMainModel(): ModelBase = (Client.getMinecraft().renderManager.getEntityRenderObject<net.minecraft.entity.Entity>(entity) as RendererLivingEntity).getMainModel()
```

Comment:
We should have an `MCEntity` alias in [`TypeAliases.kt`](https://github.com/ChatTriggers/ChatTriggers/blob/master/src/main/kotlin/com/chattriggers/ctjs/utils/kotlin/TypeAliases.kt) so you don't have to write out the full package name